pkg/resourceinterpreter/default/native/aggregatestatus.go

```diff
@@ -213,6 +213,25 @@ func aggregateJobStatus(object *unstructured.Unstructured, aggregatedStatusItems
 		return object, nil
 	}
 
+	jobFinished := false
+	for _, c := range job.Status.Conditions {
+		if c.Type == batchv1.JobComplete {
+			if c.Status == corev1.ConditionTrue {
+				jobFinished = true
+				break
+			}
+		}
+	}
+	if jobFinished {
+		now := metav1.Now()
+		job.Status.Conditions = append(job.Status.Conditions, batchv1.JobCondition{
```

Comment:
This might not be the correct fix because when karmada-apiserver is 1.30.x, karmada-controller-manager will complain :  `status.conditions: Invalid value: cannot set SuccessCriteriaMet to NonIndexed Job 和 status.conditions: Invalid value: cannot set SuccessCriteriaMet=True for Job without SuccessPolicy，`  (the reason is the feature-gate JobSuccessPolicy is false )